### PR TITLE
[TASK] API enhance JSON response

### PR DESCRIPTION
**Pull request by juergen-venne**
Get individual response for multi-file/folder-operations.

### CODE REVIEW

File: Classes/Controller/AjaxController.php [hunk 1]
```diff
@@ -318,31 +325,33 @@ public function copyResourcesAction(ServerRequestInterface $request): JsonRespon
...
         foreach ($identifiers as $identifier) {
+            $state = FileOperationResult::FAILED;
+            $message = '';
+            $resultEntity = null;
             try {
-                $sourceObject = $resourceFactory->getObjectFromCombinedIdentifier($identifier);
-                $message = '';
-                if ($resultFolder
+                $sourceObject
+                    = $resourceFactory->getObjectFromCombinedIdentifier($identifier);
```
Comment (benjaminkott) [inline Classes/Controller/AjaxController.php]: please no line break here

File: Classes/Controller/AjaxController.php [hunk 2]
```diff
@@ -364,9 +373,12 @@ public function moveResourcesAction(ServerRequestInterface $request): JsonRespon
             $targetFolderIdentifier
                 = $request->getQueryParams()['targetFolderIdentifier'];
             if (empty($identifiers)) {
-                throw new ControllerException('Identifier needed', 1553699828);
+                throw new ControllerException('Identifiers needed', 1553699828);
             }
-            if (empty($conflictMode) || !in_array($conflictMode, ['replace', 'cancel', 'rename'], true)) {
+            if (empty($conflictMode)
+                || !in_array($conflictMode, ['replace', 'cancel', 'rename'],
+                    true)
```
Comment (benjaminkott) [inline Classes/Controller/AjaxController.php]: why linebreak?

File: Classes/Controller/AjaxController.php [hunk 3]
```diff
@@ -388,31 +400,33 @@ public function moveResourcesAction(ServerRequestInterface $request): JsonRespon
...
         foreach ($identifiers as $identifier) {
+            $state = FileOperationResult::FAILED;
+            $message = '';
+            $resultEntity = null;
             try {
-                $sourceObject = $resourceFactory->getObjectFromCombinedIdentifier($identifier);
-                $message = '';
-                if ($resultFolder
+                $sourceObject
+                    = $resourceFactory->getObjectFromCombinedIdentifier($identifier);
```
Comment (benjaminkott) [inline Classes/Controller/AjaxController.php]: Linebreak?

File: Classes/Controller/AjaxController.php [hunk 4]
```diff
@@ -388,31 +400,33 @@ public function moveResourcesAction(ServerRequestInterface $request): JsonRespon
...
+                if ($resultObject
                     = $sourceObject->moveTo($targetFolderObject, null,
```
Comment (benjaminkott) [inline Classes/Controller/AjaxController.php]: oh dare, why linebreak?

File: Classes/Controller/AjaxController.php (hunk 1 above)
Comment (juergen-venne) [reply Classes/Controller/AjaxController.php]: I'll fix, sorry!

File: Classes/Controller/AjaxController.php (hunk 2 above)
Comment (juergen-venne) [reply Classes/Controller/AjaxController.php]: To make it ugly :-) I'll fix.

**Pull request merged**